packs/equipment/crafters-eyepiece.json

```
@@ -40,6 +40,25 @@
                 "selector": "crafting",
                 "type": "item",
                 "value": 1
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "system.crafting.repairValue",
+                "value": 5
+            },
+            {
+                "key": "Note",
+                "text": "When you Repair an item, increase the Hit Points restored to 10 + 10 per proficiency rank on a success or 15 + 15 per proficiency rank on a critical success.",
+                "title": "<b>Crafter's Eyepiece:</b>",
+                "selector": "crafting-check",
```

```suggestion
                "key": "Note",
                "text": "PF2E.SpecificRule.CraftersEyepiece.Note",
                "title": "{item|name}",
                "selector": "crafting-check",
```
Add the text to re-en.json at that location. Also need to add this to the greater eyepiece too.

---

Good catch, thanks for pointing it out!